### PR TITLE
Expanded discussion of `transparent_hugepages` tuner

### PR DESCRIPTION
**Pull request by greg-szrama**
Resolves Issue: https://github.com/redpanda-data/documentation-private/issues/1318

[Preview Link](https://deploy-preview-337--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-redpanda/rpk-redpanda-tune-list/#transparent_hugepages)

### CODE REVIEW

File: modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-tune-list.adoc [hunk 1]
```diff
@@ -243,7 +243,11 @@ Configuration (yaml):
 
 === transparent_hugepages
 
-The `transparent_hugepages` tuner improves memory page caching by enabling Transparent Huge Pages (THP) for CPUs that support it. Its larger memory pages reduce the number of misses from Translation Lookaside Buffer (TLB) lookups.
+RedPanda makes excellent use of OS-level Transparent HugePages (THP) through pre-allocating memory at startup. For CPUs that support THPs, this results in improved memory page caching and reduced cache misses from Translation Lookaside Buffer (TLB) lookups.
```
Comment (Deflaimun) [inline modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-tune-list.adoc]:
```suggestion
Redpanda makes excellent use of OS-level Transparent HugePages (THP) through pre-allocating memory at startup. For CPUs that support THPs, this results in improved memory page caching and reduced cache misses from Translation Lookaside Buffer (TLB) lookups.
```

File: modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-tune-list.adoc [hunk 2]
```diff
@@ -243,7 +243,11 @@ Configuration (yaml):
...
+
+Most OSes configure THP use to `madvise` by default, meaning a process must explicitly request them. RedPanda is designed to make the required `madvise` call meaning THPs are in use under typical operation scenarios. Other possible values for THP use are `disabled` or `always`.
```
Comment (Deflaimun) [inline modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-tune-list.adoc]:
```suggestion
Most OSes configure THP use to `madvise` by default, meaning a process must explicitly request them. Redpanda is designed to make the required `madvise` call meaning THPs are in use under typical operation scenarios. Other possible values for THP use are `disabled` or `always`.
```

**Review by mattschumpert — APPROVED** [review]
I defer completely to @ballard26 on this update

**Review by ballard26 — APPROVED** [review]
LGTM

File: modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-tune-list.adoc [hunk 3]
```diff
@@ -243,7 +243,11 @@ Configuration (yaml):
 
 === transparent_hugepages
 
-The `transparent_hugepages` tuner improves memory page caching by enabling Transparent Huge Pages (THP) for CPUs that support it. Its larger memory pages reduce the number of misses from Translation Lookaside Buffer (TLB) lookups.
+Redpanda makes excellent use of OS-level Transparent HugePages (THP) through pre-allocating memory at startup. For CPUs that support THPs, this results in improved memory page caching and reduced cache misses from Translation Lookaside Buffer (TLB) lookups.
+
+Most OSes configure THP use to `madvise` by default, meaning a process must explicitly request them. Redpanda is designed to make the required `madvise` call meaning THPs are in use under typical operation scenarios. Other possible values for THP use are `disabled` or `always`.
```
Comment (micheleRP) [inline modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-tune-list.adoc]:
```suggestion
Most operating systems configure THP use to `madvise` by default, meaning a process must explicitly request them. Redpanda is designed to make the required `madvise` call, meaning THPs are in use under typical operation scenarios. Other possible values for THP use are `disabled` or `always`.
```

File: modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-tune-list.adoc [hunk 4]
```diff
@@ -243,7 +243,11 @@ Configuration (yaml):
...
+
+The `transparent_hugepages` tuner overrides the OS setting, configuring THP use to `always`. This tuner only has a practical effect if the OS is configured to `disabled` for THP use. For this reason, this tuner is disabled by default in production mode. It primarily exists to allow operators to ensure THPs are in use for their installations if there is any possibility the OS is configured to `disabled`.
```
Comment (micheleRP) [inline modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-tune-list.adoc]:
```suggestion
The `transparent_hugepages` tuner overrides the operating system setting, configuring THP use to `always`. This tuner only has a practical effect if the operating system is configured to `disabled` for THP use. For this reason, this tuner is disabled by default in production mode. It primarily exists to allow operators to ensure THPs are in use for their installations if there is any possibility the operating system is configured to `disabled`.
```

File: modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-tune-list.adoc [hunk 5]
```diff
@@ -243,7 +243,11 @@ Configuration (yaml):
 
 === transparent_hugepages
 
-The `transparent_hugepages` tuner improves memory page caching by enabling Transparent Huge Pages (THP) for CPUs that support it. Its larger memory pages reduce the number of misses from Translation Lookaside Buffer (TLB) lookups.
+Redpanda makes excellent use of OS-level Transparent HugePages (THP) through pre-allocating memory at startup. For CPUs that support THPs, this results in improved memory page caching and reduced cache misses from Translation Lookaside Buffer (TLB) lookups.
```
Comment (micheleRP) [inline modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-tune-list.adoc]:
```suggestion
Redpanda makes excellent use of operating system-level Transparent HugePages (THP) by pre-allocating memory at startup. For CPUs that support THPs, this results in improved memory page caching and reduced cache misses from translation lookaside buffer (TLB) lookups.
```

**Review by micheleRP — APPROVED** [review]
looks good, left some minor suggestions

**Pull request merged**